doc/tutorials/klondike/app/.gitignore

```diff
@@ -0,0 +1,44 @@
+# Miscellaneous
```

Comment:
.gitignore files should not be included, we already have a root gitignore

Reply:
I don't know how this file and the files you refer to in subsequent comments became visible here. They are in my work-area on my local repo and laptop computer, but I certainly never committed them. I'll investigate tomorrow,

Reply:
This commit is the culprit. I was using the GitHub Desktop app (for MacOS) to push my own local commits to remote. Unfortunately I filled in some fields and pushed a button that somehow bundled up unwanted files, added them, committed them and pushed them. Most of the files were generated by Flutter's create. A couple were my own scratch-files. My git skills are rusty and were never great, so I do not know how to erase this commit and all the spurious files from my forked Flame repo on GitHub and also my local repository. Can you help, Spydon?

commit 18f71bece67fad79b71b9185dc18101f81f0df27 (HEAD -> fix#2798)
Author: IanWadham <iandw.au@gmail.com>
Date:   Wed Oct 11 19:57:53 2023 +1100

    Fixes for issue #2798
    
    The klondike_game.dart code was not keeping all 52 playing cards under its control, as required by the design. The code and doco also had to be fixed because it was broken by world and camera changes in Flame 1.9.x.

Reply:
This .gitignore is a Flutter-generated file. We need to eradicate commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 which unintentionally added and committed it.

Reply:
The offending commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 has been reverted.